### ISSUE TITLE
Split sift and polyphen fields

### ISSUE THREAD

**Issue by DrSpez** [NCI-GDC/gdc-models#45]
New mafs are coming: https://jira.opensciencedatacloud.org/browse/DAT-987
`all_effects` now has more fields inside:
```
all_effects = {
    ‘Hugo_Symbol’: 0,
    ‘consequence_type’: 1,
    ‘aa_change’: 2,
    ‘transcript_id’: 3,
    ‘ref_seq_accession’: 4,
    ‘HGVSc’: 5,
    ‘IMPACT’: 6,
    ‘CANONICAL’: 7,
    ‘SIFT’: 8,
    ‘PolyPhen’: 9,
    ‘Transcript_Strand’: 10
}
```
We need to add them to mappings

**Comment by junjun-zhang** [comment]
The work should actually be this: https://jira.opensciencedatacloud.org/browse/DAT-1003, not this: https://jira.opensciencedatacloud.org/browse/DAT-987

**Comment by DrSpez** [comment]
Note that there are more then `PolyPhen` and `Sift` fields added, but the ticket you post says only about them
Full changelog is in `[DAT-987](https://jira.opensciencedatacloud.org/browse/DAT-987)`

**Comment by junjun-zhang** [comment]
@DrSpez the model already handles some of the new fields. The other fields that are not included in the ES model, we don't need them. For example, `strand`, `CANONICAL` etc. We have them some where else already.

**Comment by junjun-zhang** [comment]
@DrSpez, this ticket is to address [DAT-1003](https://jira.opensciencedatacloud.org/browse/DAT-1003), not [DAT-987](https://jira.opensciencedatacloud.org/browse/DAT-987) which is a bioinformatics task.

**Comment by junjun-zhang** [comment]
addressed in #49
